data/mods/gen4/abilities.ts

```diff
@@ -72,6 +72,26 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 			}
 		},
 	},
+	download: {
+		inherit: true,
+		onStart(pokemon) {
+			let activate = false;
+			let totaldef = 0;
+			let totalspd = 0;
+			for (const target of pokemon.foes()) {
+				if (target.volatiles.substitute) continue;
+				activate = true;
+				totaldef += target.getStat('def', false, true);
+				totalspd += target.getStat('spd', false, true);
+			}
+			if (!activate) return;
```

Comment:
Not strictly needed because `totaldef` and `totalspd` will be zero in this case. (I believe this case already applies in the main code when you knock out your opponent with U-turn or Volt Switch and switch into a Download user.)